### PR TITLE
build: update Build ready to publish to AWS Code Artifact

### PR DESCRIPTION
**Pull request by sdelamo**
Gradle to 6.8.3
ready to publish to AWS Code Artifact

see: https://metadata.atlassian.net/browse/EX-4472

### CODE REVIEW

**Review by johncross — APPROVED** [review]
I'm not a gradle master but it all looks good to me. If James wants to review it as well there's quite a lot to learn! I can't merge it because I don't have write access - I think David might have to do the merge

**Comment by sdelamo** [comment]
I don't have write access either. Maybe @davidmilward ?

**Pull request merged**
